docfx/docs/guides/getting-started.md

```diff
@@ -20,7 +20,7 @@ If you're more of a visual person, here is a <a href="https://www.youtube.com/wa
 3. Create a new line underneath `Game_LowViolence    csgo_lv` and add `Game    csgo/addons/metamod`.
 4. Restart your game server.
 
-Your `gameinfo.gi` should look like <a href="images/gameinfogi-example.png" target="_blank">this</a>. Type `meta list` in your server console to see if Metamod is loaded.
+Your `gameinfo.gi` should look like <a href="https://docs.cssharp.dev/images/gameinfogi-example.png" target="_blank">this</a>. Type `meta list` in your server console to see if Metamod is loaded.
```

Comment:
Does this work if you just prefix it with / instead of the full url?

Reply:
****

> Does this work if you just prefix it with / instead of the full url?

I don't know, sorry. I just followed how the logo was done when I tried before. I don't have a local test environment for it either, so I thought I would just send it with the full path as it's 100% going to work.

Reply:
Actually it may need ../../